transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -68,6 +69,8 @@
     private volatile boolean autoClose = true;
     private volatile int writeBufferHighWaterMark = 64 * 1024;
     private volatile int writeBufferLowWaterMark = 32 * 1024;
+    private volatile WriteBufferWaterMark writeBufferWaterMark =
```

Comment:
@trustin seems duplicated too.
but I can't remove member: writeBufferHighWaterMark and writeBufferLowWaterMark due to old methods such as setWriteBufferHighWaterMark(int) can't modify WriteBufferWaterMark except I add package visible setMethod into WriteBufferWaterMark.

And if I remove member: writeBufferWaterMark, I can't keep same style to support getOption() except I return new WriteBufferWaterMark (writeBufferLowWaterMark ,writeBufferHighWaterMark ) for every getOption() call;

so here confused me. I am not sure my current implement look good. so WDYT?